pages/1_👀_View_Raw_Data.py

```diff
@@ -3,14 +3,27 @@
 
 from src.common import *
 from src.view import *
+from src.captcha_ import *
 
 params = page_setup()
-st.title("View raw MS data")
 
+#if local no need captcha
```

Comment:
Just checked will update the json as well st.session_state["controllo"] = True. so no need params["controllo"] = True, next time it will automatically true.

Reply:
k great so this means one could have just a call to a function e.g. called
captcha(st.session.state) ?